saleor/search/backends/newelastic.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import unicode_literals
```

Comment:
Why "new" in the file name? It should be probably named `elasticsearch.py`.

Reply:
Yes, you are right. It's legacy naming from time when "elasticsearch" existed as old one. I'll change name.